client/src/components/previews/OrganizationPreview.js

```diff
@@ -164,8 +166,11 @@ const OrganizationPreview = ({ className, uuid }) => {
                 {organization.childrenOrgs.map(organization => (
```

Comment:
Suggestion to rename the `map` variable from `organization` to `childOrg` to prevent confusion with the outer-scope `organization` variable.